_pages/team.html

```diff
@@ -142,20 +136,61 @@ <h5>Ricardo Quiroz </h5>
             <p>Ricardo Quiroz is a new member of the team and is a part of the outreach team.</p>
         </div>
         <div>
-            <img src="/images/team/placeholder.png">
+            <img src="/images/team/brandon_werbel.png">
             <p class="label">🖥️ Programmer 🖥️</p>
-            <h5>Brandon Werbel</h5>
+            <h5>Brandon Werbel </h5>
             <p>Brandon is a new member of the team and is a programmer.</p>
         </div>
         <div>
             <img src="/images/team/placeholder.png">
             <p class="label">🖥️ Programmer 🖥️</p>
-            <h5>Smridhi Madan</h5>
+            <h5>Smridhi Madan </h5>
```

Comment:
You're adding extra spaces at the end of a bunch of these tags, I'm not sure why, best to fix this